### PR TITLE
phase 1 of react design- Hannah Hoang 

### PR DESCRIPTION
**Pull request by hannahhoanggg**
hi

### CODE REVIEW

File: react-design/hannah-app/src/Banner.jsx [hunk 1]
```diff
@@ -0,0 +1,3 @@
+export default function Banner({ items }) {
```
Comment (RobertGardner) [inline react-design/hannah-app/src/Banner.jsx]: You're really only displaying a single item, so maybe it should be `item` instead of `items`.

**Review by RobertGardner — COMMENTED** [review]
Nice. Proceed!

File: react-design/hannah-app/src/Banner.jsx (hunk 1 above)
Comment (hannahhoanggg) [reply react-design/hannah-app/src/Banner.jsx]: Got it!

**Review by RobertGardner — APPROVED** [review]
This is great!

**Pull request merged**